salesagility/SuiteCRM
# Take into account multiple values from Date parameters

Date parameters were sending more than one value through the post array, upsetting the offset for other parameters.

Also put all the code handling the form parameters in one place, so that csv and pdf exports would be treated the same way.

## Description



Instead of sending all the needed variables crammed into one array, created one array for each variable.

## Motivation and Context

Issue #5111 

## How To Test This

1. Create a new report.
2. Add some fields to the report.
3. Add at least 2 conditions to the report.
4. 3.1. Create first condition with type "Value". Check the "Parameter" checkbox.
5. 3.2. Create second condition with type "Date". Check the "Parameter" checkbox.
6. Save this report.
7. Go to detail view and set the parameters.
8. Set the parameter from the first condition to any value.
9. Set the parameter from the second condition to type "Date" and select the values for example "Now -> - -> 8 -> Months".
10. Hit update and check the parameters work as expected

## Types of changes


### Final checklist
